src/bls_to_execution_change/mod.rs

```diff
@@ -0,0 +1,60 @@
+pub mod constants;
+pub(crate) mod operator;
+
+use regex::Regex;
+use types::{Address, BlsToExecutionChange};
+
+use crate::key_material::VotingKeyMaterial;
+
+pub fn bls_execution_change_from_mnemonic(
+    mnemonic_phrase: &[u8],
+    validator_start_index: u64,
+    validator_beacon_index: u64,
+    execution_address: &str,
+) -> (BlsToExecutionChange, VotingKeyMaterial) {
+    let (seed, _) = crate::seed::get_eth2_seed(Some(mnemonic_phrase));
+
+    let execution_addr_regex: Regex = Regex::new(r"^(0x[a-fA-F0-9]{40})$").unwrap();
```

Comment:
could probably pull the regex out as const since we're using it also for the withdrawal creds

Reply:
Done in https://github.com/ChorusOne/eth-staking-smith/pull/54/commits/a372991a6cf7d6eab306676f01148d0438545fdf